cclib/parser/logfileparser.py

```diff
@@ -207,18 +207,23 @@ def __init__(self, source, loglevel=logging.INFO, logname="Log",
         # Set the filename to source if it is a string or a list of strings, which are
         # assumed to be filenames. Otherwise, assume the source is a file-like object
         # if it has a read method, and we will try to use it like a stream.
+        self.isfileinput = False
         if isinstance(source, str):
             self.filename = source
             self.isstream = False
         elif isinstance(source, list) and all([isinstance(s, str) for s in source]):
             self.filename = source
             self.isstream = False
+        elif isinstance(source, fileinput.FileInput):
+            self.filename = source
+            self.isstream = False
+            self.isfileinput = True
         elif hasattr(source, "read"):
             self.filename = "stream %s" % str(type(source))
             self.isstream = True
             self.stream = source
         else:
-            raise ValueError
+            raise ValueError, "Unexpected source type."
```

Comment:
This is causing the Travis failure.

Reply:
Should be good to go now. I started using six, which I think will be useful in many other places in the code as we work towards 2.x